weather_mv/loader_pipeline/util_test.py

```diff
@@ -42,4 +43,17 @@ def test_no_duplicate_coordinates(self):
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
     # TODO(#106): Write tests...
-    pass
+    def assert_serializes_json(self, input):
+        try:
+            json.loads(json.dumps(input))
+            print("Successfully passed!")
+        except json.JSONDecodeError:
+            self.fail()
+
+    def test_numpy_float_becomes_float(self):
+        res = to_json_serializable_type(np.float(42))
+        self.assert_serializes_json(res)
+
+    def test_numpy_array(self):
+        res = to_json_serializable_type(np.array([1, 2, 3, 4, 5777, 7, 4252, 5426]))
```

Comment:
Please use an `assertEquals` to set assert against the expected result, in addition to this assert.